scripts/02_retrieve_metadata.py

```diff
@@ -220,7 +225,10 @@ def main():
             )
 
             configuration_space = pipeline.get_configuration_space(
-                {"is_sparse": sparse, "task": task}
+                DummyDatamanager(
+                    info={"is_sparse": sparse, "task": task},
+                    feat_type={"A": "numerical"}
```

Comment:
Should there also be a `"B": "categorical"` here? Otherwise, categorical hyperparameters will not be part of the meta-learning.